spec/integration/api/conversations_controller_spec.rb

```diff
@@ -26,22 +25,74 @@
     context "with valid data" do
       it "creates the conversation" do
         post api_v1_conversations_path, params: @conversation
-        @conversation_guid = JSON.parse(response.body)["conversation"]["guid"]
-        conversation = JSON.parse(response.body)["conversation"]
-
         expect(response.status).to eq 201
-        expect(conversation["guid"]).to_not be_nil
-        expect(conversation["subject"]).to eq @conversation[:subject]
-        expect(conversation["created_at"]).to_not be_nil
-        expect(conversation["participants"].length).to eq 2
-        conversation_service.find!(@conversation_guid)
+        conversation = JSON.parse(response.body)
+        confirm_conversation_format(conversation, @conversation, [auth.user, alice])
       end
     end
 
     context "without valid data" do
-      it "fails at creating the conversation" do
+      it "fails with empty body" do
         post api_v1_conversations_path, params: {access_token: access_token}
         expect(response.status).to eq 422
+        expect(response.body).to eq(I18n.t("api.endpoint_errors.conversations.cant_process"))
+      end
+
+      it "fails with missing subject " do
+        incomplete_convo = {
```

Comment:
I think you meant `incomplete_conversation`.

Reply:
Yeah I abbreviated.  I can make it the full name

Reply:
Yep please